### DISCUSSION TITLE
Removed spaces in code blocks

### DISCUSSION THREAD

**Discussion by Br31zh** [requarks/wiki#4112] [Error / Bug Report]
Hello,

When writing a code block, the spaces before the first line are removed in the rendered page. Code blocks are expected to be verbatim.

![2021-06-21-093445_1167x413_scrot](https://user-images.githubusercontent.com/47458355/122724420-33153800-d274-11eb-9023-72d813b87de6.png)

So when you are copy-pasting a snippet into an existing file, you have to re-indent it manually.

I’m using Wiki.js 2.5.201, with NodeJS 14 and PostgreSQL 12.6.

Thanks.

**Comment by SPyofcode** [comment]
same issue here

But I use this character `ㅤ` as another way :<

**Comment by IceflowRE** [comment]
``````
```
    <- here are spaces
```
``````

is rendered as
```
<- here are spaces
```

Wiki.js 2.5.298
